content/md/en/docs/main-docs/build/build-process.md

```diff
@@ -1,118 +1,155 @@
-- Read about Substrate's [Architecture](/main-docs/fundamentals/architecture) in order to understand how the client interacts with the runtime.
+The Rust version of the runtime is used as a regular Rust crate dependency to the Substrate node executable.
+You can see this by displaying the package dependencies for the `node-template-runtime` package.
+For example:
 
-## Compilation
+1. Open a terminal on a computer where you have the node template installed.
 
-Substrate runtimes are compiled to both native and Wasm.
-The native runtime is used as a regular Rust crate dependency to the Substrate client.
-You can see this when running `cargo tree -i node-template-runtime` from the terminal of a standard Substrate node template:
+1. Change to the root of the node template directory.
 
-```bash
-# Displays all packages that depend on the node-template-runtime package.
-node-template-runtime devhub/latest (...)
-└── node-template devhub/latest (...)
-```
+1. Display an inverted dependency graph by running the following command:
 
-But how is the Wasm runtime created?
-What really happens when `cargo build --release` is executed in the directory of a Substrate node project?
+    ```bash
+    cargo tree -i node-template-runtime
+    ```
 
-At a very high level, this command builds the project in it's local directory &mdash; including both native and Wasm binaries &mdash; with optimized artifacts, meaning that the output executable goes through some post-processing.
-These artifacts then result in the final executable program that enables launching a chain with the following command:
-`./target/release/node-template --dev`.
+    The command displays output similar to the following:
 
-During the build process, the Wasm runtime binary goes through 3 different stages, each containing the various steps.
-In the first stage of the build cycle, the initial Wasm runtime is built and embedded into the client.
-Once the node is compiled, the compressed Wasm binary is placed on-chain at the [`:code`](https://docs.substrate.io/rustdocs/latest/sp_storage/well_known_keys/constant.CODE.html) storage key and executed by the client.
+    ```bash
+    node-template-runtime devhub/latest
+    └── node-template devhub/latest
+    ```
 
-The following steps describes the entire build process:
+## Wasm runtime
 
-**A. Create the initial runtime Wasm binary**
+You probably already know that you can compile a Substrate node by running the `cargo build --release` command in the root directory for a Substrate node project.
+This command builds both the Rust and Wasm binaries for the project in the local directory and produces an **optimized** executable artifact.
+Producing the optimized executable artifact includes some post-compilation processing to result in a program that enables you to launch a blockchain that includes both runtime binaries.
```

Comment:
??

Optimized dos not mean that it "enables you to launch a blockchain that includes both runtime binaries."

Nothing about using either binary is part of the "optimization process"